package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/atom/atom/issues"
   },
   "license": "MIT",
-  "electronVersion": "3.1.9",
+  "electronVersion": "4.2.0",
```

Comment:
Isn't it more practical to use 4.2.2 to also have the latest bugfixes?

Reply:
Why not go straight to `v5.0.1`?

Reply:
It's just about being incremental. We tried to go to 3 first, but we're hitting https://github.com/atom/atom/issues/19372, so we wanted to see how we'd do in 4. Unfortunately we still don't really know.

Reply:
But yeah, we should be on the latest patch release.

Reply:
4.2.3 has been released yesterday

Reply:
And 4.2.4 has been tagged

Reply:
And 4.2.5 has been tagged now too